analysis/event_stream/implicit_funnel.sql

```diff
@@ -0,0 +1,37 @@
+with
+source as (
+  select * from analyst_collective.snowplow_events -- change this view for your analysis
+),
+steps as (
+  select 1 as stage, 'page_view' as "@event" union all
+  select 2, 'page_ping' union all
+  select 3, 'link_click'
+),
+people_events as (
+  select stage, "@user_id", "@event", min("@timestamp") as "@timestamp" from (
+    select steps.stage, source."@user_id", source."@event", source."@timestamp"
+    from source
+    join steps on source."@event" =  steps."@event"
```

Comment:
If there are 0 events, this stage will be left out. It should be included (outer join) with a count of 0 (coalesce it from null).